core/domain/exp_services.py

```diff
@@ -2834,3 +2834,69 @@ def rollback_exploration_to_safe_state(exp_id):
         caching_services.delete_multi(
             caching_services.CACHE_NAMESPACE_EXPLORATION, None, [exp_id])
     return last_known_safe_version
+
+
+def can_exploration_be_curated(exploration):
```

Comment:
Wait, why are you creating a new function for this? We already do frontend validations in the story editor via the `story_services.validate_explorations_for_story` function. Can't you use that?

Reply:
This function internally calls `exp_services.validate_exploration_for_story`.

Reply:
Done.